wolox-books/src/app/services/user.service.ts

```diff
@@ -30,8 +35,12 @@ export class UserService {
     });
     
     const url = `${this.root_url}/sessions`;
+    let service = this;
```

Comment:
you can remove this let and use this directly by using arrow functions like this:
```
.subscribe((token) => {
  service.localStorageService.setValue(service.localStorageService.SESSION_TOKEN, (<any>token).access_token);
  service.router.navigate(['auth']);
});